src/ansys/geometry/core/connection/product_instance.py

```diff
@@ -464,29 +464,41 @@ def prepare_and_start_backend(
             # Verify dotnet is installed
             import shutil
 
-            if not shutil.which("dotnet"):
-                raise RuntimeError(
-                    "Cannot find 'dotnet' command. "
-                    "Please install 'dotnet' to use the Ansys Geometry Core Service. "
-                    "At least dotnet 8.0 is required."
+            if version < 261:  # Only execute this block if version is less than 261
```

Comment:
We might have to be more thoughtful than just using the version. This part is specifically for Linux and, in most cases, users do not have AWP_ROOTXXX defined... which means that they are going into this through defining the ``ANSYS_GEOMETRY_SERVICE_ROOT``

I would do as follows:

1- Search if the binary exists - if it exists, use it. (this will only be true already above 261)
2- If the binary doesn't exist then we attempt to use dotnet to run the DLL (this will also address the below 261 request)